File: app/src/main/java/com/hgianastasio/biblioulbrav2/views/activities/SettingsActivity.java
package com.hgianastasio.biblioulbrav2.views.activities;

import android.support.v7.widget.Toolbar;

import com.hgianastasio.biblioulbrav2.R;
import com.hgianastasio.biblioulbrav2.views.activities.base.BaseActivity;

import butterknife.BindView;

/**
 * Created by heitor_12 on 06/06/17.
 */

public class SettingsActivity extends BaseActivity {
    @BindView(R.id.toolbar)
    Toolbar toolbar;


    @Override
    protected void preBind() {
        super.preBind();
    }

    @Override
    protected void postBind() {
        super.postBind();
        setSupportActionBar(toolbar);
        getSupportActionBar().setDisplayHomeAsUpEnabled(true);
        getSupportActionBar().setDisplayShowHomeEnabled(true);
    }
}

File: app/src/main/java/com/hgianastasio/biblioulbrav2/views/activities/SettingsActivity.kt
package com.hgianastasio.biblioulbrav2.views.activities

import com.hgianastasio.biblioulbrav2.views.activities.base.BaseActivity

/**
 * Created by heitor_12 on 06/06/17.
 */
class SettingsActivity : BaseActivity() {

    override fun postBind() {
        super.postBind()
        setSupportActionBar(toolbar)
        supportActionBar!!.setDisplayHomeAsUpEnabled(true)
        supportActionBar!!.setDisplayShowHomeEnabled(true)
    }
}